docs/api/python/accounts.md

```diff
@@ -1,11 +1,6 @@
 # Accounts: overview
 
-:::warning
-* We are currently updating the Python SDK and you may encounter errors when trying things out.
-* Apologies for the inconvenience, we will update the documentation in due course.
-:::
-
-The zkSync Python SDK, has a few methods to generate signature and verify messages.
+The zkSync Python SDK has a few methods that generate signature and verify messages.
```

Comment:
```suggestion
The zkSync Python SDK has a method that generates a signature and a few methods that verify messages.
```

Reply:
Yes, be specific about this otherwise they are both singular, or both plural. And if you use the plural as before, it's OK, it doesn't matter if it's just one.

Reply:
`The zkSync Python SDK has a method that generates a signature and a method that verify messages.` is correct.

Reply:
Thank you, amended :)